source/edge-server/configure.txt

```diff
@@ -134,273 +112,169 @@ EC2, you must open ports to accept incoming connections:
-- macOS 11.x and newer running on M1 or Intel 2018+ MacBook Pros
+.. _edge-server-install-and-configure:
 
-Other operating systems may be possible, but have not yet been verified.
+Install and Configure the Edge Server Instance
+----------------------------------------------
 
-.. _edge-server-hardware-requirements:
+After creating an Edge Server Service, you must create an Edge Server instance.
+Each service may have many Edge Server instances. When you create an 
+Edge Server instance, you get an installation script you can use to download
+the Edge Server code, and a token you can use to register the Edge Server
+instance. For more details, refer to :ref:`create-edge-server-instance`.
 
-Hardware Requirements
-~~~~~~~~~~~~~~~~~~~~~
+Use the installation script to set up and run the Edge Server on the host.
 
-Edge Server benchmarking is not yet complete, so minimum required hardware
-is not yet available. 
+.. include:: /includes/important-stop-tiered-sync-server.rst
 
-.. _edge-server-install-and-configure:
+.. procedure::
 
-Install and Configure the Edge Server
--------------------------------------
+   .. step:: Install the Edge Server Code
 
-Set up and run the Edge Server on the host.
+      Edge Server uses a command-line tool called ``edgectl`` to manage
+      the Edge Server instance on the host hardware. This includes an install
+      script to manage the installation and install required dependencies.
 
-.. include:: /includes/important-stop-tiered-sync-server.rst
+      When you create an Edge Server instance, you get a copy of this command. 
+      Use this command to install the command-line tool and the Edge Server 
+      code:
 
-.. procedure::
+      .. code-block:: shell
 
-   .. step:: Get the Edge Server Code
+         curl -s http://localhost:8080/edge/install.sh | bash -s -- --app-id="<INSERT-YOUR-APP-ID-HERE>" --platform=compose
 
-      Use ``wget`` to get the current Edge Server code as a ``.tar`` file:
+      - ``http://localhost:8080/edge/install.sh``: This assumes you are 
+        executing the command on the device where you want to install the
+        Edge Server code, and want to use ``localhost:8080`` as the port
+        to access the Edge Server. Change this as needed for your requirements.
+      - ``--app-id``: Provide your Edge Server App ID. For example, 
+        ``--app-id="edge-server-vtoyh"``. For more information, refer to 
+        :ref:`find-app-id`. 
+      - ``--platform``: You can use Docker Compose to install and manage your
+        Edge Server in a development environment, or Kubernetes to manage
+        Edge Server in a production environment. Valid values are either
+        ``compose`` or ``kubernetes``.
 
-      .. code-block:: shell
+      Follow the prompts to download and install the Edge Server code.
 
-         wget --content-disposition https://services.cloud.mongodb.com/api/client/v2.0/tiered-sync/package/latest
+   .. step:: Complete the Edge Server Configuration Details
```

Comment:
Not an optional step. The Public Preview uses the `edgectl configure` command to do some setup stuff, including exchanging the registration token for an API Key that the Edge Server instance uses to authenticate with Atlas.

But I get your point - I can tweak this heading since things have changed a bit.